src/app/app.module.ts

```diff
@@ -14,6 +14,11 @@ import { ProfilePage } from '../pages/profile/profile';
 import { TransitionPage } from '../pages/transition/transition';
 import { AssessmentPage } from '../pages/assessment/assessment';
 import { TimelinePage } from '../pages/timeline/timeline';
+import { TimelineComponent } from '../components/timeline/timeline';
+import { TimelineItemComponent } from '../components/timeline/timeline';
+import { TimelineTimeComponent } from '../components/timeline/timeline';
+
+import { ChartsModule } from 'ng2-charts';
 import { UserProvider } from '../providers/user/user';
 
 @NgModule({
```

Comment:
I see you went with multiple components within other components, nice. It'd be good if you could explain what's happening when we go over the PR in class! 🛩